src/app/plugins/platform/Docker/plugins/cms/Drupal8/installDrupal.ts

```diff
@@ -33,6 +20,34 @@ async function replaceIn(
   return writeFile(path, contents.replace(search, replacement));
 }
 
+/**
+ * Apply replacements to rename the web root in generated files.
+ */
+export async function renameWebRoot(
```

Comment:
Since we're exporting multiple functions from this file, it's probably time to rename this from `installDrupal` to something more indicative of what its purpose is: `installUtils.ts`  is probably the simplest option.

Reply:
I was considering pulling in some of the Docker writing functions to use the same way to clean up the main generator file more anyway, so generalizing it makes sense.

Reply:
Addressed in 9fb82f9.